### DISCUSSION TITLE
Home geofencing to limit notifications when I am home

### DISCUSSION THREAD

**Discussion by jkolko** [AxisCommunications/discussions#22] [General] [answered]
Hi - is there a way to limit notifications when I am home, such as understanding the location of GPS on my phone? Other camera providers have a "home mode" - something like this, native to the camera or as a third-party app?

Thanks -

**Comment by pataxis** [comment, marked as answer]
Hi @jkolko and thanks for the question.
I have looked for someone that could answer your question but many are on vacation at the moment and it will take some more weeks until people are back.

Meanwhile, could you describe your setup? What type of Axis devices and video management software (that sends you notifications) do you have?

**Reply by pataxis** [reply]
Hi again,

the team working with [Axis Camera Station (ACS) mobile application](https://help.axis.com/axis-camera-station-mobile-app) answered,

> This sounds like they want a way to turn notifications off temporarily in an automated way. That is not possible today but has been discussed and is something we will look into. This use case with geofencing is an interesting one to take extra consideration of.
